__tests__/app.test.js

```diff
@@ -167,5 +167,10 @@ describe("/api/articles/:article_id/comments", () => {
             expect(msg).toBe("400: bad request");
           });
       });
+    test('status: 204, should return 204 status code when given an article id that exists but has no associated comments', () => {
+        return request(app)
+        .get("/api/articles/2/comments")
+        .expect(204)
```

Comment:
Should be 200 and an empty array because we should still end something back that reflects how we would normally send this data back (in an array)